### ISSUE TITLE
only the image restoration part is available in open-source code，Content missing

### ISSUE THREAD

**Issue by qiqi1109** [pui-nantheera/DMAT#2]
train. py  is actually based on the implementation of the Turbulence Mitigation Transformer (TMT), a comparative model mentioned in the paper
The only loss function defined and used in train.exe is Charbonnier Loss
 standard_char=losses. CarbonnierLos().
 loss=standard_char (output, target).
 loss. backward().
Missing part: There is no mention of the detection loss Loss_ {detect} mentioned in the paper (such as classification loss BCE or box regression loss GIoU) in the code.
Model structure: The model defined in line 132=TMT-MS (...) is a multi-scale turbulence mitigation Transformer model.
This model receives a sequence of video frames and outputs a restored sequence of video frames. In the current train.by logic, the output is directly passed to standard_char for pixel loss calculation, without being passed to any detection head
